modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -5,61 +5,45 @@
 
 .Procedure
 
-. Switch to the *Developer* perspective and ensure that you are in the appropriate project, for example, `my-petclinic`. 
+. In the *Developer* perspective, switch to the relevant project, for example, `my-petclinic`.
 . In the *Topology* view, hover over the Spring PetClinic sample application to see a dangling arrow on the node.
-. Click and drag the arrow towards the *hippo* database in the Postgres Cluster to make a binding connection with the Spring PetClinic sample application. 
+. Drag and drop the arrow onto the *hippo* database icon in the Postgres Cluster to make a binding connection with the Spring PetClinic sample application.
 
-.. Enter the name and click *Create*.
+. In the *Create Service Binding* dialog, keep the default name or add a different name for the service binding, and then click *Create*.
 +
 .Service Binding dialog
 image::odc-sbc-modal.png[]
-
-
-Alternatively, in the *+Add* view, click the *YAML* option to see the *Import YAML* screen. Use the YAML editor and add the `ServiceBinding` resource:
-
-[source,YAML]
-----
-apiVersion: binding.operators.coreos.com/v1alpha1
-kind: ServiceBinding
-metadata:
-    name: spring-petclinic-pgcluster
-    namespace: my-petclinic
-spec:
-    services:
-    - group: postgres-operator.crunchydata.com
-      version: v1beta1
-      kind: PostgresCluster
-      name: hippo
-    application:
-      name: spring-petclinic
-      group: apps
-      version: v1
-      resource: deployments
-----
-
-A service binding request is created and the {servicebinding-title} controller projects the database service connection information into the application deployment as files by using a volume mount. After the request is successful, the application is redeployed and the connection is established.
-
++
+A service binding request is created and if the service connection request is successful, the application is redeployed and the connection is established.
++
 .Binding connector
 image::odc-binding-connector.png[]
-
++
 [NOTE]
 ====
 You can also use the context menu by dragging the dangling arrow to add and create a binding connection to an operator-backed service.
 
 .Context menu to create binding connection
 image::odc_context_operator.png[]
 ====
+
+. In the navigation menu, click *Topology*. The spring-petclinic deployment in the Topology view includes an Open URL link to view its web page.
+
+. Click the *Open URL* link.
+
+You can now view the Spring PetClinic sample application remotely to confirm that the application is now connected to the database service and that the data has been successfully projected to the application from the Percona MySQL database service.
+
+Once the application projects the application form successfully, the Service Binding Operator creates a working connection between the application and the database service.
```

Comment:
@Dhruv-Soni11 a little confusing to read ...the application form. Suggest dropping this sentence. Or just say that "The Service Binding Operator has successfully created a working connection between the application and the database service." to give a nice closure.

Reply:
Fixed.